### DISCUSSION TITLE
rsync (off-shore backup)

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#426] [Ideas] [answered]
To easily implement backups on multiple hard disks

https://forum.kuketz-blog.de/viewtopic.php?t=3118

**Comment by ghost** [comment]
@szaimen what do you think about it?

**Comment by szaimen** [comment, marked as answer]
This is already doable and documented: https://github.com/nextcloud/all-in-one#sync-the-backup-regularly-to-another-drive
